### DISCUSSION TITLE
Does Qodana work with golangci-lint?

### DISCUSSION THREAD

**Discussion by MichaelSnowden** [JetBrains/Qodana#145] [Q&A]
Hi, I have a project that currently uses golangci-lint. Is there a way to use this tool with Qodana?

**Comment by tiulpin** [comment]
Hi! We currently do not support it, but we plan to implement it!
I'm using `golangci-lint` as well a lot and would love to make it work with Qodana.

As we rely on GoLand for our inspections, it may be possible that the GoLand team implements the official integration in the future – https://youtrack.jetbrains.com/issue/GO-2220/Show-warnings-from-go-vet-golangci-lint
